ignite/src/main/resources/default-ignite-jdbc.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
does this file need to be built/deployed with the jar? or in the interpreter/ignite directory?